src/CLI/clitree/cli-xml/enable_mode.xml

```diff
@@ -75,6 +75,26 @@ limitations under the License.
             name="no debug"
             help="No Debug commands" />
         <!-- Special commands -->
+        <COMMAND name="interface" help="Interface commands"/>
```

Comment:
Since these commands are for demo only, can you comment out (or remove) commands?

Reply:
Just did.